modules/monitoring-creating-alerting-rules.adoc

```diff
@@ -20,7 +24,7 @@ In the current version of the Technology Preview, only administrators can access
 +
 [NOTE]
 ====
-The expression can only reference metrics exposed by your own services. Currently it is not possible to correlate existing cluster metrics.
+You cannot create an alerting rule if an alerting rule with the same name exists in another namespace.
```

Comment:
I'm not sure about this restriction.

Reply:
That is definitely possible. Here, a `namespace` label is enforced.

Reply:
Changed to "When you create an alerting rule, a namespace label is enforced on it if a rule with the same name exists in another namespace."